tuist/tuist
# Add additional scheme diagnostic options

Resolves https://github.com/tuist/tuist/issues/5212

### Short description 📝

Add support of additional scheme diagnostic options.

### How to test the changes locally 🧐

Use  .enableAddressSanitizer, .enableDetectStackUseAfterReturn, .enableThreadSanitizer 
cases of SchemeDiagnosticsOption for enabling Address Sanitizer, Detect use of stack after return, Thread Sanitizer options.

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/testing-strategy) for reference).

### Contributor checklist ✅


### Reviewer checklist ✅
